components/grid/selection/cells.md

```diff
@@ -0,0 +1,214 @@
+        public string CompanyName { get; set; }
+        public string ContactName { get; set; }
+    }
+}
+````
+
+## Selected Cells
+
+* You can get or set the selected cells through the `SelectedCells` property. The `SelectedCells` is a collection of a type `IEnumerable<GridSelectedCellDescriptor>`.
+* The `GridSelectedCellDescriptor` exposes:
+    * `SelectedCellDescriptor.ColumnField`—the [field of the associated column]({%slug components/grid/columns/bound%}#data-binding) (if provided).
+    * `SelectedCellDescriptor.ColumnId`—the [id of the associated column]({%slug components/grid/columns/bound%}#identification) (if provided).
+    * `SelectedCellDescriptor.DataItem`—the actual data item of the selected cell. It has to be casted to the Grid data model.
+* You can use the `SelectedCells` collection in two-way binding. You can predefine the selected item for your users through the two-way binding of the `SelectedCells` property. The collection will be updated by the Grid when the selection changes.
+
+### Selected Cells When Data Changes
+
+When the Grid `Data` collection changes, the `SelectedCells` collection has the following behavior:
+
+* When you update or delete a selected item in the Grid, you have to make the same in the `SelectedCells` collection through the Grid [editing events]({%slug components/grid/editing/overview%}).
+* When you create an item in the Grid, and you want to select a cell from it with its creation, you should use the Grid [editing events]({%slug components/grid/editing/overview%}).
+
+### Selected Cells Equals Comparison
+
+The `SelectedCells` collection is compared against the Grid `Data` collection in order to determine which cells will be highlighted. The default behavior of the framework is to compare objects by their reference.
+
+When the `SelectedCells` are obtained from a different data source to the Grid (e.g., from a separate service method and not from the view-model), the references may not match and so there will be no highlighted items. In such cases, you have to [override the `Equals` method of the underlying model class]({%slug grid-state%}#equals-comparison) so that it matches them, for example, by a unique identifier rather than by reference so that two objects can be equal regardless of their origin, but according to their contents. When you are overriding the `Equals` method, it is also recommended to override the [`GetHashCode`](https://docs.microsoft.com/en-us/dotnet/api/system.object.gethashcode) method as well.
+
+## SelectedCellsChanged
+
+You can respond to the user action of selecting a new cell through the `SelectedCellsChanged` event. The `SelectedCellsChanged` event receives a collection `IEnumerable<GridSelectedCellDescriptor>`. It may have no items in it. It may have only one member (the last selected item) when the `SelectionMode` is `Single`.
+
+>caption One-way binding for SelectedCells and using the SelectedCellsChanged event
+
+````CSHTML
```

Comment:
Examples with lambda expressions usually have broken syntax highlighting. In such cases, add some comment at the top.

@* ... *@